recipes/kodoja/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+mkdir -p "$PREFIX"
```

Comment:
You'll have to use `$PREFIX/bin` as `$PREFIX` won't be on `PATH` and thus those scripts cannot be found.

Reply:
Ah! Light bulb moment - thank you.

Does the fact this passed the CircleCI Mac tests indicate a test-framework problem or platform difference?

Reply:
It also confused me. I think a test framework problem -- I'll investigate, thank for reminding me!

Reply:
Cross reference https://github.com/bioconda/bioconda-recipes/issues/7312#issuecomment-361541550